src/components/qas/questions/qaCalculator/qaCalculatorUtilities.js

```diff
@@ -64,5 +120,191 @@ export const calculateCraterDiameter = ({
+    },
+    {
+      id: 'II',
+      range: [2, 3],
+      description:
+        'Felt only by a few persons at rest, especially on upper floors of buildings.',
+    },
+    {
+      id: 'III',
+      range: [3, 3.5],
+      description:
+        'Felt quite noticeably by persons indoors, especially on upper floors of buildings. Many people do not recognize it as an earthquake. Standing motor cars may rock slightly. Vibrations similar to the passing of a truck.',
+    },
+    {
+      id: 'IV',
+      range: [3.5, 4.5],
+      description:
+        'Felt indoors by many, outdoors by few during the day. At night, some awakened. Dishes, windows, doors disturbed; walls make cracking sound. Sensation like heavy truck striking building. Standing motor cars rocked noticeably.',
+    },
+    {
+      id: 'V',
+      range: [4.5, 5],
+      description:
+        'Felt by nearly everyone; many awakened. Some dishes, windows broken. Unstable objects overturned. Pendulum clocks may stop.',
+    },
+    {
+      id: 'VI',
+      range: [5, 5.5],
+      description:
+        'Felt by all, many frightened. Some heavy furniture moved; a few instances of fallen plaster. Damage slight.',
+    },
+    {
+      id: 'VII',
+      range: [5.5, 6.5],
+      description:
+        'Damage negligible in buildings of good design and construction; slight to moderate in well-built ordinary structures; considerable damage in poorly built or badly designed structures; some chimneys broken.',
+    },
+    {
+      id: 'VIII',
+      range: [6.5, 7.5],
+      description:
+        'Damage slight in specially designed structures; considerable damage in ordinary substantial buildings with partial collapse. Damage great in poorly built structures. Fall of chimneys, factory stacks, columns, monuments, and walls. Heavy furniture overturned.',
+    },
+    {
+      id: 'IX',
+      range: [7.5, 8],
+      description:
+        'General panic. Damage considerable in specially designed structures; well-designed frame structures thrown out of plumb. Damage great in substantial buildings, with partial collapse. Buildings shifted off foundations. Serious damage to reservoirs. Underground pipes broken. Conspicuous cracks in ground. In alluviated areas sand and mud ejected, earthquake fountains, sand craters.',
+    },
+    {
+      id: 'X',
+      range: [8, 8.5],
+      description:
+        'Most masonry and frame structures destroyed with their foundations. Some well-built wooden structures and bridges destroyed. Serious damage to dams, dikes, and embankments. Large landslides. Water thrown on banks of canals, rivers, lakes, etc. Sand and mud shifted horizontally on beaches and flat land. Rails bent slightly.',
+    },
+    {
+      id: 'XI',
+      range: [8.5, 9],
+      description:
+        'Rails bent greatly. Underground pipelines completely out of service.',
+    },
+    {
+      id: 'XII',
+      range: [9, 20],
```

Comment:
If this is an arbitrary cut off at 20 let's go ahead and replace 20 with `Infinity` so the user always gets a description regardless of what huge richter value they manufacture.  If this 20 is an intentional hard cap then ignore this coment.